lib/BackgroundJob/Tasks/ImageProcessingTask.php

```diff
@@ -162,9 +162,11 @@ public function execute(FaceRecognitionContext $context) {
 					$this->populateDescriptors($fld, $fr, $imageProcessingContext);
 				}
 
-				$endMillis = round(microtime(true) * 1000);
-				$duration = max($endMillis - $startMillis, 0);
-				$this->imageMapper->imageProcessed($image, $imageProcessingContext->getFaces(), $duration);
```

Comment:
I generally prefer something like:

> if ($imageProcessingContext === null) 
>    return true;

But is ok.. :wink:

Reply:
Sorry about this, I also like that approach, neutralizes excessive indentation and makes code more readable. Sometimes, though, people need to point me in, khm, khm... right direction:smile: Fixed in second iteration!